massa-execution-worker/src/interface_impl.rs

```diff
@@ -618,23 +618,23 @@ impl Interface for InterfaceImpl {
     /// * Expects a SECP256K1 signature in full ETH format.
     ///   Format: (r, s, v) v will be ignored
     ///   Length: 65 bytes
-    /// * Expects a public key in full ETH format.
-    ///   Length: 65 bytes
-    fn verify_evm_signature(
+    /// * Expects a public key in raw secp256k1 format.
+    ///   Length: 64 bytes
```

Comment:
We need to update the ABI tests / gas calibration accordingly

Reply:
You mean the runtime v2 ones? Are you asking me to put it in an issue or to do it now?

Reply:
It was more a note for myself to update my code, but if you could put it in an issue assigned to me it would be great to not forget!

Reply:
https://github.com/massalabs/massa-sc-runtime/issues/291